content/physical_properties/dielectric_permittivity/dielectric_permittivity_units.rst

```diff
@@ -16,7 +16,7 @@ The units for dielectric proporties and associated quantities are summarized in
 +---------------------------+-----------------------------------+------------------+
 | Dielectric Permittivity   | :math:`\varepsilon`               | F/m              |
 +---------------------------+-----------------------------------+------------------+
-| Dielectric Susceptibility | :math:`\chi_e`                    | *Unitsless*      |
+| Dielectric Susceptibility | :math:`\chi_e`                    | *Unitless*       |
```

Comment:
I fixed this typo a while ago. Also, dielectric susceptibility is defined after the consitutive relationship. I fell it is important to mention.